src/Tribe/Date_Utils.php

```diff
@@ -49,6 +48,109 @@ public static function datepicker_formats( $translate = null ) {
 		}
 
 		/**
+		 * As PHP 5.2 doesn't have a good version of `date_parse_from_format`, this is how we deal with
+		 * possible weird datepicker formats not working
+		 *
+		 * @param  string $format The weird format you are using
+		 * @param  string $date   The date string to parse
+		 *
+		 * @return string         A DB formated Date, includes time if possible
+		 */
+		public static function time_from_format( $format, $date ) {
```

Comment:
This method is more of a `datetime_from_format` rather than just `time_from_format`, right?